includes/database/DatabasePDO.php

```diff
@@ -23,7 +23,7 @@ class DatabasePDO extends Database
         'FETCH_BOTH' => PDO::FETCH_BOTH,
         'FETCH_NUM' => PDO::FETCH_NUM,
     ];
-    protected $error_supress = !WeBidDebug;
+    protected $error_supress = WeBidDebug;
```

Comment:
This is the error I get when trying to install. Parse error: syntax error, unexpected '!' in /hermes/bosnaweb07a/b1189/ipg.aucstacom/includes/database/DatabasePDO.php on line 26